src/util/system.h

```diff
@@ -336,6 +326,18 @@ class ArgsManager
      */
     std::string GetArg(const std::string& strArg, const std::string& strDefault) const;
 
+    /**
+     * Return path argument or default value
+     *
+     * @param arg Argument to get a path from (e.g., "-datadir", "-blocksdir" or "-walletdir")
+     * @param default_value Optional default value to return instead of the empty path.
+     * @return lightly normalized path if argument is set, with redundant "."
```

Comment:
> I'm not sure what's motivating the question, because I think you basically wrote this function and understand what it does.

I'm sorry about confusion. It was not my intention.
My motivation is the following.
The [`std::filesystem::path::lexically_normal`](https://en.cppreference.com/w/cpp/filesystem/path/lexically_normal) function returns a path "converted to [normal form](https://en.cppreference.com/w/cpp/filesystem/path) in its generic format". As normal form is well defined (at least within the `std::filesystem` library scope), I see no reasons for a term "lightly normalized path". It just raises a useless question about difference between a "lightly normalized path" and a "normalized path".

> What would your suggested change be?

```suggestion
     * @return normalized path if argument is set, with redundant "."
```

> Do you want me to change something about this comment?

It is really a minor, so feel free to ignore it.